generate-release-notes.py

```diff
@@ -60,28 +88,37 @@ def kind(issue):
             return "bug"
     return "other"
 
+
 def enterprise_feature(issue):
     for l in issue.labels:
         if l.name == "enterprise":
             return True
     return False
 
-def print_issues_to_file(f, issues):
-    for i in issues:
-        for note in extract_release_notes(i):
-            if enterprise_feature(i):
-                f.write(" - [Calico Enterprise] %s [#%d](%s) (@%s)\n" % (note, i.number, i.html_url, i.user.login))
+
+def print_issues_to_file(file, issues):
+    for issue in issues:
+        for note in extract_release_notes(issue):
+            if enterprise_feature(issue):
+                file.write(
+                    f" - [Calico Enterprise] {note} [#{issue.number}]({issue.html_url}) (@{issue.user.login})\n"  # pylint: disable=line-too-long
+                )
             else:
-                f.write(" - %s [#%d](%s) (@%s)\n" % (note, i.number, i.html_url, i.user.login))
-    f.write(u"\n")
+                file.write(
+                    f" - {note} [#{issue.number}]({issue.html_url}) (@{issue.user.login})\n"
+                )
+    file.write("\n")
+
 
 def calico_version():
-    v = yaml.safe_load(open('config/calico_versions.yml', 'r'))
-    return v['title']
+    v = yaml.safe_load(open("config/calico_versions.yml", "r", encoding="utf-8"))
```

Comment:
Is there a specific need to specify the encoding? Is `utf-8` not the default?

Reply:
I will rather not rely on defaults in this case